.travis.yml

```diff
@@ -47,9 +47,11 @@ script:
   - rustc -Vv
   - cargo -V
   - cargo check --verbose
-  - cargo check --verbose --features "syntax-highlight,sass"
   - cargo test  --verbose
-  - cargo test  --verbose --features "syntax-highlight,sass"
+  - cargo check --no-default-features --verbose
+  - cargo test --no-default-features --verbose
+  - cargo check --verbose --features "sass"
+  - cargo test  --verbose --features "sass"
```

Comment:
At some point we should make it so different feature combinations run as different jobs so they can run in parallel rather than series.

Granted that can get messy, particularly when we already have problems with our rustfmt and clippy jobs (see #305)

Reply:
Is this possible with travis?

Reply:
Yup, we just define more jobs in the matrix with a different variable set to control what happens in the job, maybe a `FEATURE` variable that is `--feature "sass"` vs `--no-default-features`.

Not something to include in this because it'd require a lot more experimenting and we shouldn't hold this up :)